test/runtime/Policies.go

```diff
@@ -713,7 +742,126 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 			}]
 		}]
 		}]`, ports)
-		testInvalidPolicy(script)
+		testInvalidPolicy(tooManyTCPPorts)
+	})
+
+	It("Test CIDR Limit", func() {
+		res := vm.ContainerCreate(helpers.Server, helpers.HttpdImage, helpers.CiliumDockerNetwork, "-l id.service2")
```

Comment:
Why you create another container and why you don't use the sample containers? You're are creating the sample containers in the BeforeEach but not use that. I would like to use the same containers (httpd[1-3])

Reply:
Nothing here, I was in the wrong describe :-) Sorry!

Reply:
No worries :)